art.taunoerik.tauno-serial-plotter.yml

```
@@ -0,0 +1,31 @@
+app-id: art.taunoerik.tauno-serial-plotter
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: tauno-serial-plotter.py
+
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --device=all
+
+modules:
+  - python3-requirements.json
+
+  - name: tauno-serial-plotter
+    buildsystem: simple
+    build-commands:
+      - install -D src/tauno-serial-plotter.py /app/bin/tauno-serial-plotter.py
+      - install -Dm644 -t ${FLATAPK_DEST}/share/applications/ art.taunoerik.tauno-serial-plotter.desktop
+      - install -Dm644 -t /app/share/appdata art.taunoerik.tauno-serial-plotter.appdata.xml
+      - install -Dm644 src/icons/tauno-plotter.svg /app/bin/icons/tauno-plotter.svg
+      - install -Dm644 src/icons/tauno-plotter.svg ${FLATAPK_DEST}/share/icons/hicolor/scalable/apps/tauno-plotter.svg
```

```suggestion
      - install -Dm644 src/icons/tauno-plotter.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/tauno-plotter.svg
```
when my suggestion contain a typo